src/renderer/components/swap/Swap.tsx

```diff
@@ -340,7 +349,8 @@ export const Swap = ({
     async (asset: Asset) => {
       // update `pendingSwitchAssets` state
       setPendingSwitchAssets(true)
-
+      // Reset previously stored fees
+      chainFeesORef.current = O.none
```

Comment:
Not in `reset()` ^?

Reply:
i believe no need to do it `reset` method. this `ref`'s value stores last successfully loaded fees for source asset and i suppose we need to reset it only in case `sourceAsset` was changed. `reset` method is called also when target asset is changed and there is no need to reset cached fee value i suppose